.editorconfig

```diff
@@ -11,3 +11,93 @@ indent_size = 4
 [{*.json,.*.yml}]
 indent_style = space
 indent_size = 2
+
+[*.cs]
```

Comment:
I pulled the .editorconfig from mono-repo to keep things to the general D2L style.

LMK if that any of the settings have some less than ideal behaviors for compatibility's sake that can be ignored in a greenfield project such as this.
(Already found one such rule which disabled IDE formatting at line breaks - removed it since its better to have it do so).

Reply:
I'm happy with this editor config, nice to be consistent.